README.md

```diff
@@ -207,6 +207,30 @@ If Doxygen and graphviz are available, the process can also build the documentat
 
 You can also selectively enable/disable other features (e.g., specific plugins you don't care about, or whether or not you want to build the recordings post-processor). Use the --help option when configuring for more info.
 
+### Building on FreeBSD
+	pkg install libsrtp2 \
+        libusrsctp \
+        jansson \
+        libnice \
+        libmicrohttpd \
+        libwebsockets \
+        curl \
+        opus \
+        sofia-sip \
+        libogg \
+        jansson \
+        libnice \
+        libconfig \
+        libtool \
+        gmake \
+        autoconf \
+        autoconf-wrapper \
+        glib
+
+	sh autogen.sh
+	./configure
+	gmake
+	gmake install
```

Comment:
Why is there a `gmake` here? I don't think the lines from autogen.sh on are needed at all, as they're shared for all systems (they're not there for the MacOS instructions for instance).

Reply:
gmake is for GNU make. BSDs has its own make